mkdocs.yml

```diff
@@ -76,6 +76,7 @@ nav:
     - Project: 
         - 'About the Kubescape project': 'project/about.md'
         - 'License': 'project/license.md'
+        - 'Releases': 'https://github.com/kubescape/helm-charts/releases'
```

Comment:
per our chat, I'd expect this to point to https://github.com/kubescape/kubescape/releases

Reply:
You're right, but let's do this step by step